packages/twig/src/patterns/components/pagination/pagination.wingsuit.yml

```diff
@@ -9,7 +9,8 @@ pagination:
       label: Current page
       description: current page number that the user is on, zero-indexed
       required: true
-      preview: "0"
+      preview: 0
+      escape: false
```

Comment:
What does escape do?

Reply:
That tells Drupal to not escape the preview value, otherwise the `0` is turned into a `Markup` object with value 0, which then breaks the preview since we then use that preview value in an arithmetic operation, like: `currentPage + 1`.